src/com/example/client.cljs

```diff
@@ -21,18 +20,22 @@
   (println "Initializing the app...")
   (app/set-root! app ui/Root {:initialize-state? true})
   (dr/initialize! app) ; make ready, if you want to use dynamic routing...
+  (df/load! app :current-user ui/User {:target (targeting/replace-at [:component/id :main :main/user])
+                                       :marker :load-user
+                                       :post-action (fn [{:keys [state]}]
+                                                      (let [authenticated? (get-in @state [:component/id :main :main/user :user/authenticated?])]
+                                                        (prn authenticated?)
+                                                        (if authenticated?
+                                                          (dr/change-route! app ["quiz" readiness-quiz-id])
+                                                          (dr/change-route! app ["login"]))))})
```

Comment:
I added a main component so that we could move everything out of the root component. So now we have to target `:component/id :main :main/user`.

After we make that action, inside of the `:post-action` hook we have to tell the router which route to load (fulcro wont default to a route). This is where we can check if the user has been authenticated and show the quiz form if they are. If not send them to the login screen.